src/Orleans/Serialization/SerializationManager.cs

```diff
@@ -863,13 +870,23 @@ public static object DeepCopyInner(object original)
             {
                 copy = copier(original);
                 SerializationContext.Current.RecordObject(original, copy);
+                return copy;
             }
-            else
+
+            var externalSerializer = externalSerializers.FirstOrDefault(serializer => serializer.IsSupportedType(t));
```

Comment:
@jason-bragg That's right, the first serializer (in the same order as they appear in the config) would become the serializer for the type.  

Serialization is already ambiguous, even with a single external serializer since all types marked [Serializable] can be handled by the Orleans serializer.  If a type were both [Serializable] and [Schema], which serializer would Orleans use?